app.authpass.AuthPass.yaml

```diff
@@ -0,0 +1,61 @@
+app-id: app.authpass.AuthPass
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+
+command: authpass
+finish-args:
+  # Filesystem and network access
+  - --filesystem=home
+  - --share=network
+  # Display access
+  - --socket=wayland
+  - --share=ipc
+  - --socket=fallback-x11
+  # Keyring access
+  - --talk-name=org.freedesktop.secrets
+  # OpenGL rendering
+  - --device=dri
+
+cleanup:
+  - /include
+  - /lib/debug
+
+modules:
+  - name: libkeybinder
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://github.com/kupferlauncher/keybinder/releases/download/keybinder-3.0-v0.3.2/keybinder-3.0-0.3.2.tar.gz
+        sha256: e6e3de4e1f3b201814a956ab8f16dfc8a262db1937ff1eee4d855365398c6020
+
+  - name: authpass
+    buildsystem: simple
+    # build-options:
+    #   build-args:
+    #     - --share=network
+    build-commands:
+      # - curl -L https://data.authpass.app/data/artifacts/authpass-linux-latest.tar.gz -O
+      - tar -zxf authpass-linux-latest.tar.gz
+      - cp -r authpass/* /app/
+      - chmod +x /app/authpass
+      - mkdir -p /app/bin
+      - ln -s /app/authpass /app/bin/authpass
+      - install -Dm644 app.authpass.AuthPass.png /app/share/icons/hicolor/512x512/apps/app.authpass.AuthPass.png
+      - install -Dm644 app.authpass.AuthPass.desktop /app/share/applications/app.authpass.AuthPass.desktop
+      - install -Dpm644 app.authpass.AuthPass.metainfo.xml /app/share/metainfo/app.authpass.AuthPass.metainfo.xml
+    sources:
+      - type: file
+        url: https://data.authpass.app/data/artifacts/authpass-linux-1.9.6_1904.tar.gz
```

Comment:
I think it would actually be possible by building on an aarch64 host, but cross compilation is still not possible 🤔 https://github.com/flutter/flutter/issues/74929

I've added the `flathub.json`